TDDApplication3.DTO/Constants/ApiRoutes.cs

```diff
@@ -0,0 +1,13 @@
+﻿namespace TDDApplication3.DTO.Constants
+{
+    public class ApiRoutes
+    {
+        public const string Healthy = "api/healthy";
+        public const string UserApi = "api/User";
+        public const string Id = "id";
+        public const string ControllerRouting = "api/[controller]";
+        public const string HealthyString = "Healthy";
+        public const string CreatedUrlReference = "https://localhost:44381/api/user/";
```

Comment:
i dont think this is good, you should use current base path if you need or just relative route